src/config/dynamic-theme-fixes.config

```diff
@@ -4431,6 +4431,13 @@ INVERT
 
 ================================
 
+orf.at/corona/stories/daten
```

Comment:
Is their any specific reason the path is being used? I think `orf.at` is enough.